packages/core-input/core-input.js

```diff
@@ -122,6 +124,7 @@ function ajax (input) {
   ajax.timer = setTimeout(() => { // Debounce next request 500 milliseconds
     if (!input.value) return // Abort if input is empty
     req.open('GET', url.replace('{{value}}', window.encodeURIComponent(input.value)), true)
+    req.setRequestHeader('X-Requested-With', 'XMLHttpRequest') // https://en.wikipedia.org/wiki/List_of_HTTP_header_fields#Requested-With
```

Comment:
Jeg regner med dette trengs for CSRF prevention:
https://www.owasp.org/index.php/Cross-Site_Request_Forgery_(CSRF)_Prevention_Cheat_Sheet#Protecting_REST_Services:_Use_of_Custom_Request_Headers

Greit å slenge med litt dokumentasjon på at den sender X-Requested-With

Jeg ser nå, at en nice to have feature er å kunne legge på egne headere på requesten. Som for eksempe Authrization header.

Reply:
Yupp! Egne headers vil kreve en ekstra event for å kunne sette opp men det går an å lage som en issue om behov :)

Reply:
Men dokumentasjon er bra! will fix